front-end/src/components/TranscriptUploadModal.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Modal from 'react-modal';
+
+class TranscriptUploadModal extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      file: null,
+    };
+  }
+  
+  handleFile(e) {
+    console.log(e.target.files, "File Uploaded");
+    this.setState({file: e.target.files})
+  }
+
+  handleUpload(e) {
+    let file=this.state.file
+    let formData= new FormData()
+
+    formData.append('transcript', file)
```

Comment:
This is done well! Just remember to use semicolons and const instead of let (since the variables are not reassigned)